assets/js/googlesitekit/data/create-settings-store.test.js

```diff
@@ -19,11 +19,11 @@
 /**
  * External dependencies
  */
+import fetchMock from 'fetch-mock-jest';
```

Comment:
@aaemnnosttv yeah, I believe `jest-fetch-mock` did it by modifying the global fetch. `fetch-mock-jest` doesn't work the same way, but we can set a jest global variable in `jest.config.js` then later set it equal to the `fetchMock` we import from the library. I've done this in `tests/js/setup-before-after.js` (we can't do it all in the `jest.config.js` because the jest methods the library relies on aren't available yet).

Reply:
Also I had to add `fetchMock` to the `globals` in `.eslintrc.json` because eslint wouldn't recognize the custom globals defined in jest.config.js. If there's a better way of doing this let me know.